modules/framework/galasa-parent/dev.galasa.framework.api.common/src/test/java/dev/galasa/framework/api/common/resources/TestCPSProperty.java

```diff
@@ -243,7 +243,7 @@ public void testCPSPropertyNoNameIsInvalid() throws InternalServletException{
     }
 
     @Test
-    public void testCPSPropertyNoValueIsInvalid() throws InternalServletException{
+    public void testCPSPropertyBlankValueIsInvalid() throws InternalServletException{
```

Comment:
This should say `testCPSPropertyBlankValueIsValid`

Reply:
yea

Reply:
Fixed